packages/vue/src/composables/useAuth.ts

```diff
@@ -1,43 +1,29 @@
 import { useActor } from '@xstate/vue';
-import { ref, reactive, Ref, watchEffect } from 'vue';
-import { getServiceFacade } from '@aws-amplify/ui';
+import { ref, reactive, Ref } from 'vue';
+import { AuthenticatorServiceFacade } from '@aws-amplify/ui';
 import { facade } from './useUtils';
 import { InterpretService } from '@/components';
 
 const service = ref() as Ref<InterpretService>;
-const useAuthenticatorValue = reactive({
-  ...facade,
-  send: '' as unknown,
-  state: '' as unknown,
-}) as any;
 
+/** @deprecated For internal use only */
 export const useAuth = (serv?: InterpretService) => {
   if (serv) {
     service.value = serv;
   }
   return useActor(service.value);
 };
 
-const useInternalAuthenticator = () => {
-  createValues();
-  watchEffect(() => {
-    createValues();
-  });
-  return useAuthenticatorValue;
-};
-
-export const useAuthenticator = useInternalAuthenticator;
-
-function createValues() {
-  if (!service.value) return;
+const useAuthInternal = reactive<AuthenticatorServiceFacade>({
```

Comment:
Is it a standard naming convention in Vue to prefix non-hooks with `use`?

Reply:
Yes. Here is an example of other composables with the `use` prefix. https://vueuse.org/functions.html

Reply:
Maybe I'm confused here due to my lack of understanding of Vue and `reactive`. Is `useAuthInternal` a function or an object?

Reply:
The `useAuthInternal` is a [reactive proxy object](https://vuejs.org/api/reactivity-core.html#reactive), it encapsulates state that we can share inside vue.

Reply:
To your point, I'll change the name of `useAuthInternal` to `stateAuth` so there is no confusion that `useAuthInternal` is a composable, that are normally functions. [4f31335](https://github.com/aws-amplify/amplify-ui/pull/3469/commits/4f313353e9485f12eca6e2976e3abf6378825670)